src/lib/Menu.js

```diff
@@ -586,10 +582,9 @@ const _titleBarTemplateFactory = (intl, locked) => [
       {
         label: intl.formatMessage(menuItems.changelog),
         click() {
-          openExternalUrl(
-            `${GITHUB_FERDIUM_URL}/ferdium-app/releases/tag/v${ferdiumVersion}`,
-            true,
-          );
+          window.location.href = window.location.href.includes('#/auth')
```

Comment:
This block of code/logic seems duplicated - can we move them into a utility class?